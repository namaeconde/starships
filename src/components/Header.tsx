import { Link } from 'react-router-dom'
import Avatar from '@mui/material/Avatar';

export const StarshipLogo = () => {
  return (
    <Avatar alt="Starship" src="/assets/home_logo.png" sx={{ width: 105, height: 79 }} variant="square"/>
  )
}

export default function Header() {

  return (
    <Link to="/">
      <StarshipLogo />
    </Link>
  )
}