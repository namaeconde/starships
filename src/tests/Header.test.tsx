import React from 'react';
import { BrowserRouter as Router } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Header from '../components/Header';

test('Header renders', () => {
  const component = renderer.create(<Router><Header></Header></Router>,);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
