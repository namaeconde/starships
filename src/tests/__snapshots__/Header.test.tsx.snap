// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders 1`] = `
<a
  href="/"
  onClick={[Function]}
>
  <div
    className="MuiAvatar-root MuiAvatar-square css-ivlicw-MuiAvatar-root"
  >
    <img
      alt="Starship"
      className="MuiAvatar-img css-1pqm26d-MuiAvatar-img"
      src="/assets/home_logo.png"
    />
  </div>
</a>
`;
